README.md

```
@@ -1,8 +1,8 @@
 <p align="center">
-  <a href="" rel="noopener"><img width=200px height=200px src="https://placehold.jp/000000/ffffff/200x200.png?text=Behat+steps&css=%7B%22border-radius%22%3A%22%20100px%22%7D" alt="Behat steps logo"></a>
+  <a href="" rel="noopener"><img width=200px height=200px src="logo.png" alt="Behat Steps steps logo"></a>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: a

(MD033, no-inline-html)